src/frontends/tests/frontend/shared/CMakeLists.txt

```diff
@@ -9,7 +9,6 @@ file(GLOB_RECURSE LIBRARY_HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/include/*.hpp)
 
 add_library(${TARGET_NAME} STATIC EXCLUDE_FROM_ALL ${LIBRARY_SRC} ${LIBRARY_HEADERS})
 
-add_subdirectory(gtest_main_manifest)
```

Comment:
Maybe these test should be removed from 2022.1.1 release branch only and keep it for master?

Reply:
PR for 2022.1.1 https://github.com/openvinotoolkit/openvino/pull/12294